testing/python/collect.py

```diff
@@ -1002,7 +1002,8 @@ def foo():
         """
         )
         try:
-            import filter_traceback_entry_as_str
+            # Type ignored because mypy doesn't know about this module.
+            import filter_traceback_entry_as_str  # type: ignore
```

Comment:
If we keep this I think the "# type: ignore" comment is enough, isn't it?  (also in other places)

Reply:
Do you mean the extra "Type ignored becuase ..." comment? This is a convention I came up with @asottile, to explain why the ignore is acceptable. I don't mind removing them here if you think they are redundant.

Reply:
I at least think they're useful